src/Sentry.Unity/SentryInitialization.cs

```diff
@@ -31,6 +32,12 @@ public static class SentryInitialization
         [RuntimeInitializeOnLoadMethod(RuntimeInitializeLoadType.BeforeSceneLoad)]
         public static void Init()
         {
+            if (!File.Exists($"{Application.dataPath}/Resources/{UnitySentryOptions.ConfigRootFolder}/{UnitySentryOptions.ConfigName}.json"))
```

Comment:
I'd move the common part into something like `UnitySentryOptions.GetConfigPath()` and reused it inside `SentryWindow.cs`

SentryWindow.cs
```csharp
protected string SentryOptionsAssetPath => UnitySentryOptions.GetConfigPath(SentryOptionsAssetName)
```

SentryInitialization.cs
```csharp
if(!File.Exists(UnitySentryOptions.GetConfigPath()))
```

and `UnitySentryOptions.GetConfigPath` can be written like

UnitySentryOptions.cs
```csharp
public static GetConfigPath(string? notDefaultConfigName)
  => $"{Application.dataPath}/Resources/{ConfigRootFolder}/{notDefaultConfigName ?? ConfigName}.json"
```

What do you think?

Reply:
I like it. The only problem I see is that UnitySentryOptions is centred around Resource right now and has no access to Application.dataPath and would return a relative path. But we should probably look for a different location to store options anyway.

Reply:
I like it! I'll. get right on it.